test/helpers.bash

```diff
@@ -215,21 +215,34 @@ function check_images() {
 }
 
 function start_crio_no_setup() {
-    "$CRIO_BINARY_PATH" \
-        --default-mounts-file "$TESTDIR/containers/mounts.conf" \
-        -l debug \
-        -c "$CRIO_CONFIG" \
-        -d "$CRIO_CONFIG_DIR" \
-        &>"$CRIO_LOG" &
-    CRIO_PID=$!
+    # first argument if set, start `cri-o` with cli `--imagestore`
```

Comment:
instead of this, you can use `CONTAINER_IMAGESTORE` in the test to set it at runtime

Reply:
done.